lib/main_containers.pm

```diff
@@ -82,15 +82,18 @@ sub load_host_tests_docker {
         loadtest 'containers/registry' if is_x86_64;
         loadtest 'containers/docker_compose';
     }
-    loadtest 'containers/validate_btrfs' if (is_x86_64);    # works currently only for x86_64, more are coming (poo#103977)
+    # works currently only for x86_64, more are coming (poo#103977)
+    # Expected to work for all but JeOS on 15sp4 after
```

Comment:
I don't get this comment. Why not in 15-SP4?

Reply:
check discussion here https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/13891#discussion_r772899092
it just doesnt work. and as it wasnt part of the scheduling before there is no good reason to fix it in this PR

Reply:
even with NUMDISK=2 ? ok...

Reply:
i havent checked. i presumed that this is out of scope because i was following the yaml

Reply:
as discusses it will fix in separate PR. resolved?